tools/vagrant/Vagrantfile

```diff
@@ -0,0 +1,113 @@
+  # `vagrant box outdated`. This is not recommended.
+  # config.vm.box_check_update = false
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine. In the example below,
+  # accessing "localhost:8080" will access port 80 on the guest machine.
+  # NOTE: This will enable public access to the opened port
+  # config.vm.network "forwarded_port", guest: 80, host: 8080
+
+  # Create a forwarded port mapping which allows access to a specific port
+  # within the machine from a port on the host machine and only allow access
+  # via 127.0.0.1 to disable public access
+  # config.vm.network "forwarded_port", guest: 80, host: 8080, host_ip: "127.0.0.1"
+
+  # Create a private network, which allows host-only access to the machine
+  # using a specific IP.
+  # config.vm.network "private_network", ip: "192.168.33.10"
+
+  # Create a public network, which generally matched to bridged network.
+  # Bridged networks make the machine appear as another physical device on
+  # your network.
+  # config.vm.network "public_network"
+
+  # Share an additional folder to the guest VM. The first argument is
+  # the path on the host to the actual folder. The second argument is
+  # the path on the guest to mount the folder. And the optional third
+  # argument is a set of non-required options.
+  # config.vm.synced_folder "../data", "/vagrant_data"
+
+  # Provider-specific configuration so you can fine-tune various
+  # backing providers for Vagrant. These expose provider-specific options.
+  # Example for VirtualBox:
+  #
+  # config.vm.provider "virtualbox" do |vb|
+  #   # Display the VirtualBox GUI when booting the machine
+  #   vb.gui = true
+  #
+  #   # Customize the amount of memory on the VM:
+  #   vb.memory = "1024"
+  # end
+  #
+  # View the documentation for the provider you are using for more
+  # information on available options.
+
+  # Enable provisioning with a shell script. Additional provisioners such as
+  # Puppet, Chef, Ansible, Salt, and Docker are also available. Please see the
+  # documentation for more information about their specific syntax and use.
+  config.vm.provision "shell", inline: <<-SHELL
+     apt-get update
+     apt-get update 
+	 apt-get install -y git curl bzip2 wget tar unzip xauth
+	 apt-get install -y libpng12-0 libjasper1 libopenmpi1.10 libgomp1 libxext-dev \
+	 	libxrender-dev libxslt1.1 libxtst-dev libgtk2.0-0 libcanberra-gtk-module \
+		libfontconfig1 libxrender1 libxtst6 libxi6
+	
+	# Install Intellij	
+	ed=C
+	# Fetch the most recent version
+	VERSION="173.4674.33"
+	# Prepend base URL for download
+	URL="https://www.jetbrains.com/intellij-repository/releases/com/jetbrains/intellij/idea/ideaI$ed/$VERSION/ideaI$ed-$VERSION.zip"
+	#URL="https://download.jetbrains.com/idea/ideaI$ed-$VERSION.tar.gz"
+	echo $URL
+	# Truncate filename
+	FILE=$(basename ${URL})
+	# Set download directory
+	DEST=/home/vagrant/Downloads/$FILE

```

Comment:
Is this script run as the root user or as the `vagrant` user?
Use `$HOME` instead of `/home/vagrant` ?

Reply:
yes its run as root so $HOME is /home/root